### PR TITLE
add wicked color schemes package

### PR DESCRIPTION
**Pull request by b0xw00d**

### CODE REVIEW

File: repository/c.json [hunk 1]
```diff
@@ -2067,6 +2067,17 @@
 			]
 		},
 		{
+			"name": "Color Schemes - Wicked Schemes by b0xw00d",
+			"details": "https://github.com/b0xw00d/wicked_color_schemes",
+			"labels": ["theme", "color scheme", "syntax highlighting", "wicked"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": true
```
Comment (wbond) [inline repository/c.json]: You'll need to add at least one [semver](http://semver.org) tag to https://github.com/b0xw00d/wicked_color_schemes/tags for this to show up to users.

File: repository/c.json [hunk 2]
```diff
@@ -2067,6 +2067,17 @@
 			]
 		},
 		{
+			"name": "Color Schemes - Wicked Schemes by b0xw00d",
+			"details": "https://github.com/b0xw00d/wicked_color_schemes",
+			"labels": ["theme", "color scheme", "syntax highlighting", "wicked"],
```
Comment (wbond) [inline repository/c.json]: I don't think `syntax highlighting` is correct here, unless I missed a `.tmLanguage` or `.sublime-syntax` file in the repo.

Additionally, `theme` is for Sublime Text themes that skin the whole UI.

**Comment by b0xw00d** [comment]
Labels are fixed and tag is added to the repo!  Thanks!

**Comment by b0xw00d** [comment]
I'm not missing anything else here am I?  All feedback is welcome and appreciated!  😃

**Comment by wbond** [comment]
Sorry for the delay, just hadn't reviewed packages recently.

**Pull request merged**